.gitignore

```diff
@@ -3,3 +3,4 @@ tags
 .ccls
 .vscode
 .venv
+.idea
```

Comment:
Nit: others are alphabetical :)

Could also put this in your ~/.gitignore then it will catch all repos.